fapolicy_analyzer/ui/main_window.py

```diff
@@ -358,11 +362,49 @@ def on_saveAsMenu_activate(self, menuitem, data=None):
         fcd.destroy()
 
     def on_aboutMenu_activate(self, *args):
-        aboutDialog = self.get_object("aboutDialog")
-        aboutDialog.set_transient_for(self.window)
-        aboutDialog.set_version(f"v{app_version}")
-        aboutDialog.run()
-        aboutDialog.hide()
+        rpm_path = ""
+        try:
+            rpm_path = subprocess.getoutput(
+                "rpm -ql fapolicy-analyzer | grep fapolicy-analyzer-about"
+            )
+        except Exception:
+            logging.debug("Failed to query for an rpm install")
+        abspath = (
+            rpm_path
+            if not rpm_path == ""
+            else path.abspath("./data/fapolicy-analyzer-about.json")
+        )
+        if path.isfile(abspath):
+            try:
```

Comment:
Either load the build-info file or show unknown.  We dont care about what is running on the current system, ie calling subprocess to get this stuff in real time is not interesting.